### DISCUSSION TITLE
NovelAIDiffusion繪圖版權問題、Vtuber爭議

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#790] [General]
如標題所述，AI繪圖雖然會但給我們便利，但同時也會引發繪圖師、動畫師、遊戲師、Vtuber等作者的版權問題，是現今最為嚴重的問題，但也有些Vtber啟發了創作大賽，挺有意思的。有興趣的可參考看看。

參考網址：
https://www.techbang.com/posts/100716-novelail-ai-painting
https://news.gamebase.com.tw/news/detail/99403336

**Comment by ghost** [comment]
我認為AI繪圖最大的問題就是他訓練時所用的圖庫，如果這些圖庫都是經過授權的買下的，那其AI繪製出的圖應該不會有版權問題，這些只是繪畫上使用的技術面的不同而已，就像抽象派畫風一樣，未來說不定會有ai派的畫風呢~

**Comment by Pietra1226** [comment]
如果AI有畫作比賽，那評審是不是也要用AI
